addons/mail/static/src/components/rtc_controller/rtc_controller.xml

```diff
@@ -3,79 +3,81 @@
 
     <t t-name="mail.RtcController" owl="1">
         <div class="o_RtcController">
-            <t t-if="thread">
-                <t t-if="isCurrentActiveCall"><t t-set="phoneTitle">Disconnect</t></t>
-                <t t-else=""><t t-set="phoneTitle">Join Call</t></t>
-                <div class="o_RtcController_thread">
-                    <span class="o_RtcController_button o_RtcController_call_button"
-                        t-att-class="{ 'o-active': isCurrentActiveCall }"
-                        t-att-aria-label="phoneTitle"
-                        t-att-title="phoneTitle"
-                        t-on-click="_onClickPhone">
-                        <i class="fa fa-lg fa-phone o_RtcController_thread_phone"/>
+            <t t-if="env.mailRtc">
+                <t t-if="thread">
+                    <t t-if="isCurrentActiveCall"><t t-set="phoneTitle">Disconnect</t></t>
+                    <t t-else=""><t t-set="phoneTitle">Join Call</t></t>
+                    <div class="o_RtcController_thread">
+                        <span class="o_RtcController_button o_RtcController_call_button"
+                            t-att-class="{ 'o-active': isCurrentActiveCall }"
+                            t-att-aria-label="phoneTitle"
+                            t-att-title="phoneTitle"
+                            t-on-click="_onClickPhone">
```

Comment:
Would prefer `>` on newline, so that parent/child is more discernible at a glance. Applies to at least this whole template

Reply:
This was just an indentation change but I guess I'll take the opportunity to clean the template a bit.

Reply:
Actually it's too much work to do it now, it should fundamentally be rewritten later anyway.